src/style.css

```diff
@@ -0,0 +1,136 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.card {
+  width: 200px;
+  padding-top: 32px;
+  padding-right: 16px;
+  padding-bottom: 16px;
+  padding-left: 16px;
+  background: #ffff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__content {
+  font-weight: 500;
+  color: #060b35;
+  margin-top: 40px;
+  width: 166px;
+  height: 36px;
+  line-height: 18px;
+}
+
+.card__product_code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  width: 98px;
+  height: 14px;
+  margin-top: 4px;
+  align-self: flex-start;
+  text-align: center;
+  display: flex;
+  align-items: center;
+}
+
+.card__reviews {
+  display: flex;
+  width: 166px;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.